db_stress_tool/multi_ops_txns_stress.h

```diff
@@ -370,5 +393,38 @@ class InvariantChecker {
                 "MultiOpsTxnsStressTest::Record::c_ must be 4 bytes");
 };
 
+class MultiOpsTxnsStressListener : public EventListener {
+ public:
+  explicit MultiOpsTxnsStressListener(MultiOpsTxnsStressTest* stress_test)
+      : stress_test_(stress_test) {
+    assert(stress_test_);
+  }
+
+#ifndef ROCKSDB_LITE
+  ~MultiOpsTxnsStressListener() override {}
+
+  void OnFlushCompleted(DB* db, const FlushJobInfo& info) override {
+    assert(db);
+#ifdef NDEBUG
```

Comment:
ifdef not necessary, even if the `(void)db` is.

(I'm not sure db_stress even makes sense to run in release mode, considering the number of `assert(false)` etc.)

Reply:
It also depends on SyncPoint, so maybe we really don't need `ifdef NDEBUG`.